osu.Game/Rulesets/Difficulty/Skills/StrainSkill.cs

```diff
@@ -57,7 +57,25 @@ public sealed override void Process(DifficultyHitObject current)
                 currentSectionEnd += SectionLength;
             }
 
-            currentSectionPeak = Math.Max(StrainValueAt(current), currentSectionPeak);
+            double strain = StrainValueAt(current);
+            currentSectionPeak = Math.Max(strain, currentSectionPeak);
+
+            // Store the strain value for the object
+            ObjectStrains.Add(strain);
+        }
+
+        /// <summary>
+        /// Calculates the number of strains weighted against the top strain.
+        /// The result is scaled by clock rate as it affects the total number of strains.
+        /// </summary>
+        public double CountDifficultStrains()
```

Comment:
```suggestion
        public virtual double CountDifficultStrains()
```

following earlier comment